test/e2e/framework/network/utils.go

```diff
@@ -336,8 +336,6 @@ func (config *NetworkingTestConfig) GetEndpointsFromContainer(protocol, containe
 func (config *NetworkingTestConfig) DialFromNode(protocol, targetIP string, targetPort, maxTries, minTries int, expectedEps sets.String) {
 	var cmd string
 	if protocol == "udp" {
-		// TODO: It would be enough to pass 1s+epsilon to timeout, but unfortunately
```

Comment:
now , it seems that we use the https://github.com/kubernetes/kubernetes/pull/88370/files#diff-85e02e1f07138a061f89e5550f36290fL84 image。